src/main/java/com/appraisal/repositories/EmployeeManagerRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.appraisal.repositories;
+
+import com.appraisal.entities.EmployeeManager;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
You don't need this. Spring will automatically scan this as long as it extends JpaRepository/CrudRepository